actions/access_keys/search.go

```diff
@@ -17,19 +17,14 @@ import (
 // @Description	Search access key
 // @Tags		Access-key
 // @Produce		json
-// @Param		page query int false "page"
-// @Param		page_size query int false "page_size"
-// @Param		order_by_field query string false "order_by_field"
-// @Param		sort_direction query string false "sort_direction"
-// @Param		metadata query string false "metadata"
-// @Param		conditions query string false "conditions"
+// @Param		SearchRequestParameters body actions.SearchRequestParameters false "SearchRequestParameters model containing metadata, conditions and query params"
```

Comment:
Maybe something like:
```suggestion
// @Param		SearchRequestParameters body actions.SearchRequestParameters false "Supports targeted resource searches with filters for metadata and custom conditions, plus options for pagination and sorting to streamline data exploration and analysis"
```
For all search requsts body

Reply:
Done